bin/create-ebs-volume

```diff
@@ -288,37 +288,45 @@ function create_and_attach_volume() {
 
     logthis "created volume: $volume_id [ $volume_opts ]"
 
-    # In theory this shouldn't need to loop as aws ec2 wait will retry but I have seen it exceed request limits
-    for i in {1..3} ; do
-       if aws ec2 wait volume-available --region $region --volume-ids $volume_id; then
-         logthis "volume $volume_id available"
-         break
-       fi
-    done
+    # Check for volume availability until ready or $max_attempts
+    local VOLUME_STATE="none"
+    local VOLUME_STATE_RETRIES=0
+	while [ "$VOLUME_STATE" != "available" ] ; do 
+        if [[ $VOLUME_STATE_RETRIES -le $max_attempts ]]; then
```

Comment:
Using `aws ec2 wait` to check for volume availability is less likely to run into API request limits at scale